android/build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
 }
 
 dependencies {
-	implementation ('io.socket:socket.io-client:1.0.0') {
+	implementation ('io.socket:socket.io-client:2.0.1') {
```

Comment:
2.1.0 :)

Reply:
thats a newer version. Will do a quick test with it to see if nothing changed.